Firestore/core/src/remote/grpc_connection.h

```diff
@@ -109,6 +112,8 @@ class GrpcConnection {
 
   void RegisterConnectivityMonitor();
 
+  void AddFirebasePlatformLoggingHeader(grpc::ClientContext& context) const;
```

Comment:
This declaration does not seem to have a definition (but perhaps I missed it)? It also seems not to have any uses.

Reply:
Fixed, sorry.